src/components/site/BreadCrumbLinkItem.js

```diff
@@ -1,12 +1,14 @@
 import React from "react";
 import PropTypes from "prop-types";
+import classnames from "classnames";
 
 const BreadcrumbLinkItem = props => {
-  const { link, text } = props;
+  const { link, text, className } = props;
+  const BreadcrumbClasses = classnames("breadcrumb", className);
```

Comment:
@theilman7 "breadcrumb" assigned by se?

Reply:
Yep, `.breadcrumb` is a class assigned to all children by SE.